torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4782,13 +4785,14 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            dtypes=all_types_and(torch.float16, torch.bfloat16, torch.bool),
            sample_inputs_func=sample_inputs_max_min_binary,),
     OpInfo('nn.functional.hardswish',
+           aten_name="hardswish",
            supports_autograd=True,
            assert_autodiffed=True,
            sample_inputs_func=sample_inputs_hardswish,
            dtypesIfCUDA=floating_types_and(torch.half, torch.bfloat16),
            supports_gradgrad=False,
            supports_out=False,
-           autodiff_fusible_nodes=["aten::hardswish"]),
+           autodiff_nonfusible_nodes=["aten::hardswish"]),
```

Comment:
why was this change made ?

Reply:
fusible means it's inside a fusion group inside a differentiable graph and nonfusible means inside a differentiable graph

Reply:
@eellison the test wasn't running previously so I think @Krovatkin is just correcting the metadata